folium/plugins/marker_cluster.py

```diff
@@ -26,53 +33,54 @@ class MarkerCluster(Layer):
     icon_create_function : string, default None
         Override the default behaviour, making possible to customize
         markers colors and sizes.
-
-    locations: list of list or array of shape (n, 2).
-        Data points of the form [[lat, lng]].
-    popups: list of length n.
-        Popup for each marker.
-    icons: list of length n.
-        Icon for each marker.
+    options : dict, default None
```

Comment:
We should look into a docstring template functionality to avoid these repetitions.
I believe pandas has a nice example of that, I'll look into it.

Reply:
Good idea. I don't know how to do that yet, but I'm interested to know.